WinUIGallery/WinUIGallery.DesktopWap.Package.wapproj

```diff
@@ -44,8 +44,8 @@
     <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
     <EntryPointProjectUniqueName>WinUIGallery.DesktopWap.csproj</EntryPointProjectUniqueName>
     <AppxBundleNameForOutput>WinUIGallery.DesktopWap</AppxBundleNameForOutput>
-    <WindowsPackageType>Desktop</WindowsPackageType>
-    <!-- it will create packaged desktop app -->
+    <WindowsPackageType>Desktop</WindowsPackageType> <!-- it will create packaged desktop app -->
+    <WindowsAppSdkSelfContained>true</WindowsAppSdkSelfContained>
```

Comment:
always?

Reply:
my bad. this got slipped in. I will remove it